app/views/arf_reports/_list.html.erb

```diff
@@ -10,7 +10,7 @@
   <% for arf_report in @arf_reports %>
     <tr>
       <td><%= name_column(arf_report.host) %></td>
-      <td><%= _("%s ago") % time_ago_in_words(arf_report.date) %></td>
+      <td><%= _("%s ago") % time_ago_in_words(arf_report.reported_at) %></td>
       <td><%= report_arf_column(arf_report.passed, "label-info") %></th>
       <td><%= report_arf_column(arf_report.failed, "label-danger") %></th>
       <td><%= report_arf_column(arf_report.othered, "label-warning") %></th>
```

Comment:
in [line 21](https://github.com/theforeman/foreman_openscap/pull/135/files#diff-71e7815f8e3d6c41d1c1a504ae6266f4L21) should be `arf_report.host` instead of `arf_report.asset.host`